autoload/lsp.vim

```diff
@@ -298,15 +298,28 @@ function! s:ensure_init(buf, server_name, cb) abort
         return
     endif
 
+    if has_key(l:server_info, 'capabilities')
+        let l:capabilities = l:server_info['capabilities']
```

Comment:
when would you want to manually set this? Is this when you want to disable certain feature of the lang server? Just curious.